rust/src/rfb/rfb.rs

```diff
@@ -122,6 +129,10 @@ impl RFBState {
         }
     }
 
+    fn set_event(tx: &mut RFBTransaction, event: RFBEvent) {
```

Comment:
Could this be directly implemented in tx and not in state ?

Reply:
But it is. See https://github.com/OISF/suricata/blob/c2ee17130da59cfbd09c416f8d401ffe2d7e58ac/rust/src/rfb/rfb.rs#L98

I think this is a comment on an outdated code base: 
![image](https://github.com/OISF/suricata/assets/27732/a1a7b64b-fd51-4b5d-825b-84478a508d8a)